digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/RoadAddressLinkBuilder.scala

```diff
@@ -35,10 +35,7 @@ class RoadAddressLinkBuilder(roadwayDAO: RoadwayDAO, linearLocationDAO: LinearLo
     val roadName = roadAddress.roadName
     val municipalityCode = roadLink.attributes.getOrElse(MunicipalityCode, 0).asInstanceOf[Number].intValue()
     val municipalityName = municipalityNamesMapping.getOrElse(municipalityCode, "")
-    val administrativeClass = roadAddress.administrativeClass match {
-      case AdministrativeClass.Unknown => getAdministrativeClass(roadLink.administrativeClass, UnknownLinkType)
-      case _ => roadAddress.administrativeClass
-    }
+    val administrativeClass = roadLink.administrativeClass
```

Comment:
This might now broke the original logic where the roadaddress admin class was preferred over the link's admin class...